src/panel/details.tsx

```diff
@@ -40,6 +45,25 @@ interface DetailsProps { result: Result, height: IObservableValue<number> }
 
         const {result, height} = this.props;
         const helpUri = result?._rule?.helpUri;
+        const renderItem = (location: Location) => {
```

Comment:
>renderItem [](start = 14, length = 10)

can we rename this to renderLocation for consistency with renderStack?

Reply:
Fixed